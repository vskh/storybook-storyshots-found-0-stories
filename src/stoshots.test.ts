import testStorySnapshots from "@storybook/addon-storyshots";

testStorySnapshots();