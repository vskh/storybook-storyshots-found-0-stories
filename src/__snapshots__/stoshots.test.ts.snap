// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Demo story Default 1`] = `
<div>
  Hello
</div>
`;
